### PR TITLE
added four level 1 spells

### PR DESCRIPTION
**Pull request by Isaac-Tait**
I added the following:

`2020-10-13-spells-1-absorb-elements.md`
`2020-10-13-spells-1-acid-stream.md`
`2020-10-13-spells-1-arcane-weapon.md`
`2020-10-13-spells-1-armor-of-agathys.md`

I would like to contribute more but I wanted to make sure I was completing the spells correctly. 

Will you go in later and change the descriptions and other descriptors (e.g. school, casting time, duration, etc...)? Or is that something I should do? If so where can I conduct the research for the various spells? 

Thank you for your time. I look forward to your response.

### CODE REVIEW

**Comment by magicalmusings** [comment]
All of the descriptors should be filled out before being merged. If you look up 'dnd spells 5e' on google, plenty of resources should pop up. One website I like is this one for the data http://dnd5e.wikidot.com/spells

Hopefully that clears up your question, let me know if there's anything else I can clarify

**Comment by Isaac-Tait** [comment]
That does clear up my question. Thank you. I will get to work and submit a new PR

**Review by magicalmusings — CHANGES_REQUESTED** [review]
Per our conversation just now, verify aspects of spells from http://dnd5e.wikidot.com/spells for the spells you'd like to contribute prior to next PR / approval.

**Comment by Isaac-Tait** [comment]
I added 12 level 1 spells with proper formatting and all fields updated/completed as requested.

File: _posts/spells/first-level/2020-10-13-spells-1-absorb-elements.md
Comment (magicalmusings) [inline _posts/spells/first-level/2020-10-13-spells-1-absorb-elements.md]: 1 reaction, which you take when you take acid, cold, fire, lightning or thunder damage

File: _posts/spells/first-level/2020-10-13-spells-1-acid-stream.md
Comment (magicalmusings) [inline _posts/spells/first-level/2020-10-13-spells-1-acid-stream.md]: Self (30 foot-line)

File: _posts/spells/first-level/2020-10-13-spells-1-acid-stream.md
Comment (magicalmusings) [inline _posts/spells/first-level/2020-10-13-spells-1-acid-stream.md]: Concentration, up to 1 minute

File: _posts/spells/first-level/2020-10-13-spells-1-armor-of-agathys.md
Comment (magicalmusings) [inline _posts/spells/first-level/2020-10-13-spells-1-armor-of-agathys.md]: V, S, M (a cup of water)

File: _posts/spells/first-level/2020-10-13-spells-1-arms-of-hadar.md
Comment (magicalmusings) [inline _posts/spells/first-level/2020-10-13-spells-1-arms-of-hadar.md]: First Level instead of Cantrips for categories

File: _posts/spells/first-level/2020-10-13-spells-1-beast-bond.md
Comment (magicalmusings) [inline _posts/spells/first-level/2020-10-13-spells-1-beast-bond.md]: V, S, M (a bit of fur wrapped in cloth)

File: _posts/spells/first-level/2020-10-13-spells-1-beast-bond.md
Comment (magicalmusings) [inline _posts/spells/first-level/2020-10-13-spells-1-beast-bond.md]: up to 10 minutes

File: _posts/spells/first-level/2020-10-13-spells-1-cause-of-fear.md
Comment (magicalmusings) [inline _posts/spells/first-level/2020-10-13-spells-1-cause-of-fear.md]: Cause Fear

File: _posts/spells/first-level/2020-10-13-spells-1-ceremony.md
Comment (magicalmusings) [inline _posts/spells/first-level/2020-10-13-spells-1-ceremony.md]: Evocation (ritual)

File: _posts/spells/first-level/2020-10-13-spells-1-ceremony.md
Comment (magicalmusings) [inline _posts/spells/first-level/2020-10-13-spells-1-ceremony.md]: ***Atonement***

File: _posts/spells/first-level/2020-10-13-spells-1-ceremony.md
Comment (magicalmusings) [inline _posts/spells/first-level/2020-10-13-spells-1-ceremony.md]: For all these headings like Wedding, put them in Bold and Italics (i.e. * * * )

File: _posts/spells/first-level/2020-10-13-spells-1-chaos-bolt.md
Comment (magicalmusings) [inline _posts/spells/first-level/2020-10-13-spells-1-chaos-bolt.md]: Put d8 into table on Markdown

Example table: 
| d8 | damage type | 
|:--:|:--------------:|
|1 | Acid |

File: _posts/spells/first-level/2020-10-13-spells-1-chromatic-orb.md
Comment (magicalmusings) [inline _posts/spells/first-level/2020-10-13-spells-1-chromatic-orb.md]: M (Diamond worth at least 50 gp)

**Review by magicalmusings — CHANGES_REQUESTED** [review]
Went ahead and added some comments to each file for edits, other than that things look good and I'll approve once those comments are resolved.

File: _posts/spells/first-level/2020-10-13-spells-1-arms-of-hadar.md
Comment (Isaac-Tait) [reply _posts/spells/first-level/2020-10-13-spells-1-arms-of-hadar.md]: Updated all to reflect

**Comment by Isaac-Tait** [comment]
Requested changes have been made. Thank you for the review, some of the descriptors vary from spells list page to when you actually click on the spell name. I will keep that in mind for future work. Cheers.

**Review by magicalmusings — APPROVED** [review]

**Comment by magicalmusings** [comment]
Fantastic, thank you so much @Isaac-Tait ! Appreciate your help with this.

**Pull request merged**
